dist/components/tabs/stencil-tabs.js

```diff
@@ -0,0 +1,121 @@
+$(document).ready(function() {
+    var init = function() {
+        // Set up initial state for accessibility
+        var $tabButtons = $('.c-tabs__button');
```

Comment:
For true components, there’s no need for `js-` classes. They are useful when layering behaviour over arbitrary markup (so that you can evolve the markup, its styles and the behaviour “module” separately. But for a functional unit like a component that’s not a problem, and then the prerogative is “keep it simple”.